src/view/draggable/focus-on-drag-handle.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
A new file to help us to get the draggable ref which is currently not provided by consumers